win/rl/pynethack.cc

```diff
@@ -177,14 +177,25 @@ class Nethack
         }
     }
 
+    void
+    set_spawn_monsters(bool spawn_monsters)
+    {
+#ifdef NLE_ALLOW_CONTROL
+        init_settings_.spawn_monsters = spawn_monsters;
+        use_init_settings = true;
```

Comment:
I think this has the side effect of changing the way seeding works, which isn't necessarily what we want here.